### DISCUSSION TITLE
Enhanced Error Reporting

### DISCUSSION THREAD

**Discussion by benhaynes** [directus/directus#3073] [Feature Requests]
Ability to automatically submit errors to a central Directus Server so that the core team can better triage the most important issues/bugs/errors.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
